packages/api/AlvTimeWebApi/Controllers/InvoiceRateController.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Threading.Tasks;
+using AlvTime.Business.InvoiceRate;
+using AlvTimeWebApi.Controllers.Utils;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace AlvTimeWebApi.Controllers;
+
+[Route("api/user")]
+[ApiController]
+public class InvoiceRateController : Controller
+{
+
+    private readonly RetrieveUsers _userRetriever;
+    private readonly InvoiceRateService _invoiceRateService;
+
+    public InvoiceRateController(RetrieveUsers userRetriever, InvoiceRateService invoiceRateService)
+    {
+        _userRetriever = userRetriever;
+        _invoiceRateService = invoiceRateService;
+    }
+
+    [HttpGet("InvoiceRate")]
+    [Authorize(Policy = "AllowPersonalAccessToken")]
+    public async Task<decimal> FetchUserInvoiceRate(DateTime? fromDate, DateTime? toDate)
+    {
+        var user = _userRetriever.RetrieveUser();
+
+        if (!toDate.HasValue)
+        {
+            toDate = DateTime.Now;
+        }
+
+        if (!fromDate.HasValue)
+        {
+            fromDate = DateTime.Now.AddDays(-30);
+        }
+
+        return await _invoiceRateService.GetEmployeeInvoiceRateForPeriod(user.Id, fromDate.Value, toDate.Value);
```

Comment:
Gikk du bort fra å returnere en liste med måneder med rate for hver?

Reply:
Statistikk er neste steg. Dette er bare for tallet oppe i høyre hjørne.

Reply:
Tar statistikk i neste PR :smile: